inst/include/population_dynamics/population/population.hpp

```diff
@@ -33,15 +33,387 @@
 
 #include "../../common/model_object.hpp"
 #include "subpopulation.hpp"
+#include "../recruitment/recruitment.hpp"
+#include "../maturity/maturity.hpp"
+#include "../growth/growth.hpp"
+#include "../fleet/fleet.hpp"
 
 namespace fims {
+  /*TODO: 
+  Review, add functions to evaluate, push vectors back to fleet (or point to fleet directly?)
+*/
+    /**
+     * @brief Population class. Contains subpopulations
+     * that are divided into generic partitions (eg. sex, area).
+     */
+    template <typename Type>
+    struct Population : public FIMSObject<Type> {
+        static uint32_t id_g; /*!< reference id for population object*/
+        size_t nyears; /*!< total number of years in the fishery*/
+        size_t nseasons; /*!< total number of seasons in the fishery*/
+        size_t nages; /*!< total number of ages in the population*/
+        size_t nfleets; /*!< total number of fleets in the fishery*/
+        //constants
+        const double proportion_female = 0.5; /*!< Sex proportion fixed at 50/50 for M1*/
 
-/**
- * Population class. Contains subpopulations
- * that are divided into generic partitions (eg. sex, area).
- */
-template <typename T>
-struct Population : FIMSObject<T> {};
-}  // namespace fims
+        // parameters are estimated; after initialize in create_model, push_back to parameter list - in information.hpp (same for initial F in fleet)
+        std::vector<Type> log_naa; /*!< estimated parameter: log numbers at age*/
+        std::vector<Type> log_Fmort; /*!< estimated parameter: log Fishing mortality*/
+        std::vector<Type> log_M; /*!< estimated parameter: log Natural Mortality*/
+        std::vector<Type> log_q; /*!< estimated parameter: log catchability*/
+        
+        //Transformed values
+        std::vector<Type> naa; /*!< transformed parameter: numbers at age*/
+        std::vector<Type> Fmort; /*!< transformed parameter: Fishing mortality*/
+        std::vector<Type> M; /*!< transformed parameter: Natural Mortality*/
+        std::vector<Type> q; /*!< transformed parameter: catchability*/
+        
+        std::vector<Type> ages; /*!< vector of the ages for referencing*/
+        std::vector<Type> mortality_M; /*!< vector of natural mortality by year and age*/
+        std::vector<Type> mortality_F; /*!< vector of fishing mortality by year and age*/
+        std::vector<Type> mortality_Z; /*!< vector of total mortality by year and age*/
+
+        //derived quantities
+        std::vector<Type> weight_at_age; /*!< Derived quantity: expected weight at age */
+        // fecundity removed because we don't  need it yet?
+        std::vector<Type> numbers_at_age; /*!< Derived quantity: population expected numbers at age in each year*/
+        std::vector<Type> unfished_numbers_at_age; /*!< Derived quantity: population expected unfished numbers at age in each year*/
+        std::vector<Type> catch_at_age;/*!< Derived quantity: catch at age*/
```

Comment:
The unit testing group is removing the catch_at_age variable because it is never used.

Reply:
This should also be removed on line 126 in the initialize function.